Src/Fitnet/Shared/BusinessRulesEngine/IBusinessRule.cs

```diff
@@ -0,0 +1,7 @@
+namespace SuperSimpleArchitecture.Fitnet.Shared.BusinessRulesEngine;
+
+internal interface IBusinessRule
```

Comment:
```suggestion
public interface IBusinessRule
```
BusinessRule is a well-defined interface that can be used by several modules.  I think this is a candidate for public interface. 

What do you think about that :D ?

Reply:
In the first round I would like to keep it internal (as we just have 1 project) - the goal is to have as less questionable things as possible